code/game/gamemodes/shadowling/thrall_abilities.dm

```diff
@@ -0,0 +1,121 @@
+
+/obj/effect/proc_holder/spell/targeted/sling/lesser_glare //a defensive ability, nothing else. can't be used to stun people, steal tasers, etc. Just good for escaping
+	name = "Lesser Glare"
+	desc = "Makes a single target dizzy for a bit."
+	panel = "Thrall Abilities"
+	charge_max = 450
+	human_req = TRUE
+	clothes_req = FALSE
+	action_icon = 'icons/mob/actions/actions_shadowling.dmi'
+	action_icon_state = "glare"
+	thrall_ability = TRUE
+
+/obj/effect/proc_holder/spell/targeted/sling/lesser_glare/InterceptClickOn(mob/living/caller, params, atom/t)
+	. = ..()
+	if(iscarbon(t))
+		var/mob/living/carbon/target = t
+		if(target.stat)
+			to_chat(user, "<span class='warning'>[target] must be conscious!</span>")
+			revert_cast()
+			return
+		if(is_shadow_or_thrall(target))
+			to_chat(user, "<span class='warning'>You cannot glare at allies!</span>")
+			revert_cast()
+			return
+		user.visible_message("<span class='warning'><b>[user]'s eyes flash a bright red!</b></span>")
+		target.visible_message("<span class='danger'>[target] suddendly looks dizzy and nauseous...</span>")
+		if(in_range(target, user))
+			to_chat(target, "<span class='userdanger'>Your gaze is forcibly drawn into [user]'s eyes, and you suddendly feel dizzy and nauseous...</span>")
+		else //Only alludes to the thrall if the target is close by
+			to_chat(target, "<span class='userdanger'>Red lights suddenly dance in your vision, and you suddendly feel dizzy and nauseous...</span>")
+		target.confused += 25
+		target.Jitter(50)
+		if(prob(25))
+			target.vomit(10)
+	else
+		revert_cast()
+
+////////////////////
+////THRALL GUISE////
+////////////////////
+
+/obj/effect/proc_holder/spell/self/lesser_shadow_walk //Thrall version of Shadow Walk, only works in darkness, doesn't grant phasing, but gives near-invisibility
+	name = "Guise"
+	desc = "Wraps your form in shadows, making you harder to see."
```

Comment:
This ability was kind of useless, especially on a massive two-minute cooldown. I don't know what to replace it with, though.